src/containers/Movie/MovieInfo/Lists/Popover/component.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { Button } from 'antd'
+import { map } from 'lodash'
+import PropTypes from 'prop-types'
+
+import CreateUserListModal from 'Containers/CreateUserListModal'
+
+const Popover = ({
+  userLists,
+  addMovieToExistingList,
+  createListModalOpen,
+  onToggleModal,
+  actionModal,
+  modalLoading,
+  openModalHandler
+}) => (
+  <>
+    <div>
+      <Button
+        type="link"
+        onClick={openModalHandler}
+      >
+        Create new list ...
+      </Button>
+    </div>
+    <div className="popup-user-lists">
+      {map(userLists, ({ description, id }) => (
+        <div key={id}>
+          <Button
+            onClick={() => addMovieToExistingList(id)}
```

Comment:
do not inline event handlers. collection rendering is a good place for extracting item to separate component